### DISCUSSION TITLE
增加一个搜索功能？

### DISCUSSION THREAD

**Discussion by wizardbyron** [timqian/openprompt.co#24] [Ideas]
可以搜索到 Prompt

**Comment by timqian** [comment]
感谢建议，搜索是很有必要的

**Comment by timqian** [comment]
增加了一个搜索框
- 可以直接对话
- 可以搜索 starred prompts(为了鼓励 star 有用的 prompts)

<img width="1297" alt="Screen Shot 2023-03-26 at 20 38 08" src="https://user-images.githubusercontent.com/5512552/227776239-fbdd67dc-aacd-4a33-b54b-5eb39818cfd5.png">
